### DISCUSSION TITLE
Highlighting an area in QGraphicsView PyQt5

### DISCUSSION THREAD

**Discussion by Aleksandr34nov** [qutebrowser/qutebrowser#8289] [Q&A]
Hi, everybody
I am writing an interface in PyQt 5
It is necessary to implement an area selection mode on the QGraphicsView on which the QGraphicsScene is located
Example on gif:
![Example](https://github.com/user-attachments/assets/a8becf60-2dda-4597-afc8-b63e3b566311)

It is necessary to be able to resize the selected area using points in the middle of the edges and corners of the area.
The area is highlighted in the uploaded photo.
That is, you just need to select an area and get the coordinates of the selected area.

If someone can suggest how this can be done or video similar solutions, then I will be grateful for any sources and links to examples.
Thank you in advance.

**Comment by The-Compiler** [comment]
It's beyond me why you think it's okay to spam multiple repositories with this that have nothing at all to do with your question.

Other one for context: https://github.com/python-qt-tools/PyQt5-stubs/issues/215
